general/jit/jit.hpp

```diff
@@ -0,0 +1,190 @@
+// Copyright (c) 2010-2023, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#ifndef MFEM_JIT_HPP
+#define MFEM_JIT_HPP
+
+#include "../../config/config.hpp"
+
+#define MFEM_JIT // prefix to label JIT kernels, arguments or includes
+
+#ifdef MFEM_USE_JIT
+
+#include <vector>
+#include <string>
+#include <iomanip>
+#include <sstream>
+#include <iostream>
+#include <functional>
+#include <unordered_map>
+
+namespace mfem
+{
+
+/**
+ * @brief The Jit structure provides the folowing static functions:
```

Comment:
Since JIT is an abbreviation, it makes sense for the name of the class to be all caps: `JIT`.

There is also a typo in the comments.

```suggestion
 * @brief The Jit structure provides the folowing static functions:
```

Reply:
@tzanio, the above suggestion does not change anything.

Reply:
`done`